_includes/neurips/2019/presenters-info.nips.2019.html

```diff
@@ -0,0 +1,28 @@
+ <!-- Presenters Modals -->
+ {% for post in site.tags.oral %}
```

Comment:
tag reference will need to be specific to this workshop and year for example: 
{%  for post in site.tags.oral-neurips2019 %}

Reply:
I don't have data for the presenters ; I think they are sending this information so I'm going to wait before I do this change since right now is empty all the presenters info